hgtv/views/video.py

```diff
@@ -39,21 +39,25 @@ def process_video(video, new=False):
         if parsed.netloc in ['youtube.com', 'www.youtube.com']:
             try:
                 video_id = parse_qs(parsed.query)['v'][0]
-                r = requests.get('https://gdata.youtube.com/feeds/api/videos/%s?v=2&alt=json' % video_id)
-                jsondata = r.json() if callable(r.json) else r.json
-                if jsondata is None:
+                r = requests.get('https://www.googleapis.com/youtube/v3/videos?part=snippet&id={video_id}&key={api_key}'.format(
+                    video_id=video_id, api_key=app.config.get('YOUTUBE_API_KEY', '')
+                ))
+                try:
+                    jsondata = r.json() if callable(r.json) else r.json
+                except ValueError as e:
+                    raise DataProcessingError("Unable to parse video data, please try after sometime")
```

Comment:
Also log the error. Call `app.logger.error` with the content of the request so we can see what happened. [API](https://docs.python.org/2/library/logging.html#logging.Logger.error).